packages/vue/src/examples/pages/home/Home.vue

```diff
@@ -209,157 +209,269 @@ export default {
+          }
+        },
+        imageD: {
+          mobile: {
+            url: "/assets/storybook/Home/imageD.jpg",
+            content: "angelina_trn"
+          },
+          desktop: {
+            url: "/assets/storybook/Home/imageB.png",
+            content: "@instagram"
+          }
+        }
+      }
     };
   },
   methods: {
     toggleWishlist(index) {
-      this.products[index].isOnWishlist = !this.products[index].isOnWishlist;
+      return (this.products[index].isOnWishlist = !this.products[index]
+        .isOnWishlist);
     }
   }
 };
 </script>
 <style lang="scss" scoped>
 @import "~@storefront-ui/vue/styles";
-#home {
+.home {
+  display: flex;
+  flex-direction: column;
   box-sizing: border-box;
-  @include for-desktop {
-    max-width: 1240px;
-    margin: 0 auto;
-  }
-}
-.section {
-  padding: 0 var(--spacer-big);
-  @include for-desktop {
-    padding: 0;
-  }
-}
-.banner-grid {
-  margin: var(--spacer-big) 0;
-  @include for-desktop {
-    margin: var(--spacer-extra-big) 0;
+  padding: 0 var(--spacer-sm);
+  &__hero {
+    display: flex;
+    order: 1;
+    margin: var(--spacer-xl) 0 0 0;
   }
-}
-.banner-central {
-  @include for-desktop {
-    --banner-container-flex: 0 0 70%;
+  &__heading {
+    display: flex;
+    order: 2;
+    justify-content: space-between;
+    margin: var(--spacer-base) 0;
+    &__title {
+      font-size: var(--h3-font-size);
+      font-weight: var(--font-medium);
+      font-family: var(--font-family-secondary);
+    }
   }
-}
-.call-to-action {
-  margin: var(--spacer-big) 0;
-  @include for-desktop {
-    margin: calc(var(--spacer-extra-big) * 2) 0;
+  &__carousel {
+    --image-height: 16.125rem;
+    display: flex;
+    justify-content: center;
+    order: 3;
   }
-}
-.product-carousel {
-  margin: 0 calc(var(--spacer-big) * -1) 0 0;
-  @include for-desktop {
-    margin: var(--spacer-big) 0;
-    --carousel-padding: var(--spacer-big);
-    --carousel-max-width: calc(100% - 13.5rem);
+  &__banner-grid {
+    display: flex;
+    flex-direction: column;
+    order: 4;
+    margin: var(--spacer-xl) 0 var(--spacer-base) 0;
   }
-}
-.product-card {
-  max-width: unset;
-  &:hover {
-    --product-card-box-shadow: 0 4px 20px rgba(168, 172, 176, 0.19);
+  &__call-to-action {
+    display: flex;
+    order: 5;
   }
-}
-.images-grid {
-  max-width: 60rem;
-  margin: 0 auto;
-  &__row {
+  &__section-instagram-feed {
     display: flex;
-    & + & {
-      margin: calc(var(--spacer-big) / 2) 0 0 0;
-      @include for-desktop {
-        margin: var(--spacer-big) 0 0 0;
-      }
-    }
+    flex-direction: column;
+    order: 6;
+    margin: var(--spacer-2xl) 0 var(--spacer-xl) 0;
   }
-  &__col {
-    flex: 1;
-    margin: 0;
-    & + & {
-      margin: 0 0 0 calc(var(--spacer-big) / 2);
-      @include for-desktop {
-        margin: 0 0 0 var(--spacer-big);
-      }
+  &__instagram-feed {
+    --image-width: 100%;
+    --image-height: 100%;
+    display: grid;
```

Comment:
are you test it on IE?

Reply:
If you mean edge it looks fine 
![image](https://user-images.githubusercontent.com/32042425/78291435-ab83c200-7525-11ea-8a59-c36aaa4a7e8b.png), but I found another bug:  
![image](https://user-images.githubusercontent.com/32042425/78291620-fc93b600-7525-11ea-926b-21c05e52108d.png)